app/controllers/api/projects_controller.rb

```diff
@@ -152,6 +152,11 @@ def find_project_as_json_with_dependencies!(platform, name, version_name, subset
       version.update_column(:dependencies_count, deps.size)
     end
 
+    if version.dependencies_count.nil? && deps.empty? && version.updated_at < 1.day.ago
+      version.touch
+      PackageManagerDownloadWorker.perform_async(platform, name, version.number)
+    end
```

Comment:
I just realized we'd never hit this bc of line 151-153 right now. I think we might be able to replace those lines with your stanza?

Reply:
woops, nm, I just realized the difference between these two. Could combine them into the same `if version.dependencies_count.ni?` block?